core/src/main/java/org/openedx/core/data/model/CourseDates.kt

```diff
@@ -0,0 +1,157 @@
+package org.openedx.core.data.model
+
+import com.google.gson.annotations.SerializedName
+import org.openedx.core.CourseDatesBadge
+import org.openedx.core.utils.TimeUtils
+import java.util.Date
+
+data class CourseDates(
+    @SerializedName("dates_banner_info") val datesBannerInfo: CourseDatesBannerInfo,
```

Comment:
Please make sure to pass only those params that are going to be used in the future.

Reply:
for now we can retain all.